easybuild/tools/testing.py

```diff
@@ -58,6 +59,48 @@
 
 _log = fancylogger.getLogger('testing', fname=False)
 
+_exclude_env_from_report = []
+DEFAULT_EXCLUDE_FROM_REPORT = [
+    'KEY',
+    'SECRET',
+    'TOKEN',
+    'PASSWORD',
+    'API',
+    'AUTH',
+    'CREDENTIALS',
```

Comment:
```suggestion
    'CREDENTIAL',
```
Might as well cover all scenarios